modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/podcast/EpisodeManagerImpl.kt

```diff
@@ -355,6 +355,24 @@ class EpisodeManagerImpl @Inject constructor(
         }
     }
 
+    override suspend fun updateAutomaticallyCachedStatus(
+        episode: BaseEpisode?,
+        automaticallyCached: Boolean,
+    ) {
+        if (episode == null || episode is UserEpisode) return
```

Comment:
I find it extremely strange to accept `BaseEpisode?` when the only type we can work with is `UserEpisode`. I know this is a convention in the project, where we accept as inputs overly broad types, and I'm not requesting a change here. Just something that stands out to me.